Library/Homebrew/utils/diskusage.rb

```diff
@@ -0,0 +1,57 @@
+require "find"
+
+module Utils
+  class DiskUsage
```

Comment:
I don't think this needs to be a separate class? Could this be part of `Pathname.abv`?